src/modules/BuilderPlace/actions/builderPlace.ts

```diff
@@ -392,6 +393,36 @@ export const getBuilderPlaceById = async (id: string) => {
   }
 };
 
+
+export const getDiscordWebhookDetailsByBuilderPlaceId = async (id: string) => {
```

Comment:
for maintanability, i think it's good to not create a new get function everytime we want some info on a BP. We can use a single getById that returns all the datas